mcgj/mcgj.py

```diff
@@ -28,7 +28,15 @@ def index():
 @bp.route("/profile")
 @login_required
 def profile():
-    return render_template("edit_profile.html", user=current_user)
+    user_tracks_query = "SELECT * FROM tracks WHERE user_id = ? ORDER BY cue_date DESC LIMIT 50"
```

Comment:
What if there's no tracks with a cue_date? It's probably best to exclude those...